scripts/src/java/org/oppia/android/scripts/todo/data/Todo.kt

```diff
@@ -0,0 +1,9 @@
+package org.oppia.android.scripts.todo.data
+
+/**
+ * Represents the structure of TODO.
+ * @property filePath the path of the file
```

Comment:
Nit: blank doc line before the property list (like we do for ``@param``s).

Reply:
Added.